CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGED] Use .UTC timezeone everywhere we use time.Unix. #7066
+* [CHANGED] Federation: It does not use remote storages anymore as designed initially. #7077
+* [CHANGED] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
+* [CHANGED] Moved to Go 1.14.2. #7100
```

Comment:
That is not relevant for end users

Reply:
Not something I plan to argue about, but with my user hat on, that has been relevant several times. Much of my initial need for help in 2015 was around Go and the version needed to compile, not around Prometheus proper.

Reply:
Users don't compile prometheus (?)